src/Tablet/hoc/Aux.js

```diff
@@ -0,0 +1,4 @@
+//only returns the children of the props (arguments passed)
+const aux = props => props.children;
```

Comment:
Can you introduce us to the idea for this? What problems is this function solving?